LoginForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+
+const LoginForm = ({ onSubmit }) => {
+  return (
+    <Formik
+      initialValues={{ email: '', password: '' }}
+      onSubmit={onSubmit}
+    >
+      <Form>
+        <Field type="email" name="email" placeholder="Email" required />
+        <Field type="password" name="password" placeholder="Password" required />
+        <button type="submit">Login</button>
+      </Form>
```

Comment:
Consider adding a loading state to the login button. This will provide better feedback to the user, especially during slow network connections.